dcpy/metadata/validate.py

```diff
@@ -150,12 +150,16 @@ def validate_csv(csv_path: Path, dataset: models.Dataset, metadata: models.Metad
 
 
 def validate_package(package_path: Path, metadata: models.Metadata):
+    errors = []
```

Comment:
FYI, validate_package was throwing errors for non-csv types, bc we were returning None.